tartiflette/sdl/transformers/schema/field.py

```diff
@@ -0,0 +1,58 @@
+from collections import OrderedDict
+from typing import Optional, Union, Dict
+
+from tartiflette.sdl.transformers.schema import Name, GraphQLDefinition, \
+    GraphQLArgumentDefinition
+
+
+class GraphQLFieldDefinition(GraphQLDefinition):
```

Comment:
Is the directives will be stored in that class? _(these related to the field)_

Reply:
Unclear yet